### DISCUSSION TITLE
Where did Dynamic Prompts go in the UI?

### DISCUSSION THREAD

**Discussion by tusharbhutt** [adieyal/sd-dynamic-prompts#169] [Q&A] [answered]
Dynamic Prompts has disappeared from the UI as of this morning!

Seems to be some new setup in AUTYO1111, has anyone else encountered this? 

![image](https://user-images.githubusercontent.com/26264462/211206867-bfdfcf8d-3545-4113-a196-45efae26e510.png)

It's showing as active:

![image](https://user-images.githubusercontent.com/26264462/211206890-a4046ff4-f003-460a-8da0-bc5b7639bc72.png)

But when running a prompt, the system looks to the stable-diffusion-webui-wildcards folder, not extensions\sd-dynamic-prompts\wildcards

**Comment by adieyal** [comment]
Is there an error message in the terminal? If it doesn't appear, it often means that something broke while it was loading

**Reply by tusharbhutt** [reply]
it is back, not sure if it's because of an update pull (there was an update to 2.2.2 or if it's because the settings in AUTO1111 now allow for UI section modification and I added dynamic_prompts here:

![image](https://user-images.githubusercontent.com/26264462/211207577-39abd466-1308-4db5-84f7-9363eadc9272.png)

Probably the former :)
 
I do have a potential bug though, will post in the issues section if it has not been corrected

**Comment by tusharbhutt** [comment, marked as answer]
Closing this as not an issue
